### PR TITLE
Use Toolkit v4

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Update framework to use terra-toolkit v4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,9 +48,9 @@
     "postinstall:project": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
     "postpublish": "npm run dependency-markdown && git add --all && git commit -m 'Updated DEPENDENCIES.md files'",
-    "start": "tt:serve",
-    "start-heroku": "tt:serve-static --site build ",
-    "start-static": "tt:serve-static --config webpack.config",
+    "start": "tt-serve",
+    "start-heroku": "tt-serve-static --site build ",
+    "start-static": "tt-serve-static --config webpack.config",
```
Comment (Blackop778) [inline package.json]: The config argument should be removed as it causes serving statically to fail

File: package.json [hunk 2]
```diff
@@ -48,9 +48,9 @@
...
     "test": "npm run jest && npm run wdio",
     "wdio": "wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js",
     "tds:generate-app-config": "tds:generate-app-config"
```
Comment (Blackop778) [inline package.json]: This needs to be changed to `tds-generate-app-config` to align with the terra-dev-site update

Comment (emilyrohrbough) [reply package.json]: good catch! I bet I missed these in each repo 😭

File: package.json [hunk 3]
```diff
@@ -48,12 +48,12 @@
...
-    "start": "tt:serve",
-    "start-heroku": "tt:serve-static --site build ",
-    "start-static": "tt:serve-static --config webpack.config",
+    "start": "tt-serve",
+    "start-heroku": "tt-serve-static --site build ",
+    "start-static": "tt-serve-static",
     "test": "npm run jest && npm run wdio",
     "wdio": "wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js",
-    "tds:generate-app-config": "tds:generate-app-config"
+    "tds:generate-app-config": "tds-generate-app-config"
```
Comment (mjhenkes) [inline package.json]: Delete this line

**Pull request merged**